server/app/controllers/dev/seeding/DevDatabaseSeedController.java

```diff
@@ -134,6 +134,12 @@ public Result clearCache() {
     return ok("The cache has been cleared");
   }
 
+  /**
+   * Clear cache if it is enabled in settings.
+   *
+   * <p>Note: this is not safe to do in most deployed instances, because there may be multiple
```

Comment:
**question:** Is it worth protecting this operation with a condition like [this](https://github.com/civiform/civiform/blob/e85e7fc39062cd294e9481ceca59771edbd37c53/server/app/controllers/dev/seeding/DevDatabaseSeedController.java#L61)?

Reply:
Good call out - both callers of this function call that method, so I left it out here, but do you think its worth adding here as well? This is a private function, so it would just be new callers within this file that would need to add a call to the isDevOrStaging method. I can lean either way :)

Reply:
My suggestion is to put it here as well. A bit of extra defensive coding to protect against future refactors where the existing checks may get lost.

Reply:
Yeah that makes sense to me! Updated